test/unit/CMakeLists.txt

```diff
@@ -57,3 +57,11 @@ target_link_libraries(proj_test_unit
   gtest
   ${PROJ_LIBRARIES})
 add_test(NAME proj_test_unit COMMAND proj_test_unit)
+
+add_executable(pj_phi2_test
```

Comment:
I'm wondering if we need to create a dedicated test program for that ? Seems a bit overkill. We could put that as an extra source of the existing test. Same for automake

Reply:
I think we should start with separate targets.  Keep test files as isolated from each other as possible.  A test could accidentally mess up errno, the default ctx, default logging or other state.  In fact tests should be in there that do that kind of thing.